### PR TITLE
React Form Controls | Yang An Boada

### PR DESCRIPTION
**Pull request by yanganboada**
form

### CODE REVIEW

File: react-form-controls/src/index.jsx [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+  handleClick(e) {
+    e.preventDefault();
+  }
+
+  render() {
+    return (
+      <form>
+        <label>
+          Email:
+          <input type='text' name='Email' onChange={this.handleChange} />
```
Comment (scbowler) [inline react-form-controls/src/index.jsx:27]: Here you need to have a `value` attribute set with your value for the input in state, in this case `this.state.val`

**Review by scbowler — CHANGES_REQUESTED** [review]
Looking good, but one update needed. Let me know if you have any questions.

File: react-form-controls/src/index.jsx [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+    this.handleChange = this.handleChange.bind(this);
+    this.handleClick = this.handleClick.bind(this);
+  }
+
+  handleChange(e) {
+    this.setState({ val: e.target.value });
+  }
+
+  handleClick(e) {
+    e.preventDefault();
```
Comment (scbowler) [inline react-form-controls/src/index.jsx:19]: Here you should console log your state

**Review by scbowler — CHANGES_REQUESTED** [review]
One more thing I missed...

File: react-form-controls/src/index.jsx (hunk 1 above)
Comment (yanganboada) [reply react-form-controls/src/index.jsx]: Updated

File: react-form-controls/src/index.jsx (hunk 2 above)
Comment (yanganboada) [reply react-form-controls/src/index.jsx]: Updated

**Pull request merged**